Pages/Cart-Page/cart.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://enztron-dev-branch.netlify.app/components.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+
+    <link rel="icon" type="image/x-icon" href="../../Images/favicon-icon.png">
+    <link rel="stylesheet" href="../../styles.css">
+    <link rel="stylesheet" href="cart.css">
+    <script defer src="../topnavbar.js"></script>
+    <title>Enztron Store | Cart</title>
+</head>
+<body>
+    <div class="top-bar">
+        <div class="left-topbar-container">
+            <button class="icon-btn"><i class="fa fa-bars" aria-hidden="true"></i></button>
+            <h2 id="top-bar-brand-name">Enztron Book Store</h2>
+            <div class="search-bar">
+                <input class="search-bar-input" placeholder="Search"/>
```

Comment:
font-family: inherit;
For the input, font-family can be inherited to make the font-style consistent throughout the page.